tool-data/q2view_display.loc.sample

```diff
@@ -0,0 +1,3 @@
+# Table used for listing simple Intermine display servers
+#<unique_id>	<display_name>	<url>
+main_q2view	qiime2view	https://view.qiime2.org/visualization/?type=html&src=%(q2view_file_url_qp)s
```

Comment:
Will `q2_view_file_url_qp` be URI encoded?

Reply:
Yes, but that happens in the .xml file. 

We can also add additional servers by adding new lines here.